tests/providers/google/cloud/operators/test_gcs_system.py

```diff
@@ -22,26 +22,49 @@
 from tests.test_utils.gcp_system_helpers import CLOUD_DAG_FOLDER, GoogleSystemTest, provide_gcp_context
 
 
+@pytest.fixture(scope="module")
+def helper():
+    return GcsSystemTestHelper()
```

Comment:
Can we give this fixture a more descriptive name?

Reply:
Do you have any proposition? Looks fine for me, but I can change it, if you want. For example, I can change it to `gcs_helper`.

Reply:
@uranusjr @deedmitrij I cannot resolve the issue cause I'm not the author of the PR. Please resolve or answer or guide me what changes should I do.

Reply:
`gcs_helper` sounds good to me. The PR looks good to me besides this as well (and I already approved), so let’s wait for another maintainer to take a look and merge if they feel it’s good to go.

Reply:
Helper is as good as it can get. I also do not like general "Util, Helper, Manager, Processor" but if you look at Airflow code we have quite a bunch of those everywhere (and as much as we fight it - there ARE cases where they are needed and actually usually in tests, I'd be far more relaxed to have those).